### PR TITLE
chore(gatsby-admin): add user information to readme

### PR DESCRIPTION
**Pull request by mxstbr**

### CODE REVIEW

File: packages/gatsby-admin/README.md [hunk 1]
```diff
@@ -1,8 +1,28 @@
...
+
+To enable Admin for your local Gatsby site, run `gatsby develop` with the `GATSBY_EXPERIMENTAL_ENABLE_ADMIN` environment variable enabled:
+
+```sh
+GATSBY_EXPERIMENTAL_ENABLE_ADMIN=true gatsby develop
+```
+
+You can then visit `localhost:8000/___admin` to view Admin for your Gatsby site!
+
+![Screenshot 2020-10-02 at 16 03 08 (1)](https://user-images.githubusercontent.com/7525670/95580804-36df9200-0a38-11eb-80a7-fbd847a13da1.png)
```
Comment (ascorbic) [inline packages/gatsby-admin/README.md:15]: Coudl you give these better names

**Comment by mxstbr** [comment]
Done! 👍

File: packages/gatsby-admin/README.md [hunk 2]
```diff
@@ -1,8 +1,28 @@
 # Gatsby Admin
 
-A visual interface to configure your Gatsby site.
+A visual interface to configure your Gatsby site. Currently alpha testing.
 
-## Architecture
+## Getting started
+
+To enable Admin for your local Gatsby site, run `gatsby develop` with the `GATSBY_EXPERIMENTAL_ENABLE_ADMIN` environment variable enabled:
```
Comment (LekoArts) [inline packages/gatsby-admin/README.md:7]: To lower the bar to entry we could/should make this more explicit. See the intro section of https://www.gatsbyjs.com/docs/file-system-page-creation/ which explains that you add this to your package.json and that Windows users should use `cross-env`

People already have their existing Gatsby sites and `scripts` in package.json, let's give them instructions to change that :)

File: packages/gatsby-admin/README.md [hunk 3]
```diff
@@ -1,8 +1,28 @@
...
+You can then visit `localhost:8000/___admin` to view Admin for your Gatsby site!
+
+![Gatsby Admin homepage showing a list of installed plugins, as well as a search input to search for plugins to install](https://user-images.githubusercontent.com/7525670/95580804-36df9200-0a38-11eb-80a7-fbd847a13da1.png)
+
+When you select an installed plugin or search for a new one, you will be able to install/uninstall/configure it:
+
+![Gatsby Admin plugin page showing the README of the gatsby-plugin-sitemap, as well as an input field to configure the options for said plugin](https://user-images.githubusercontent.com/7525670/95580764-27f8df80-0a38-11eb-9f26-8a2cbbc4b07d.png)
+
+Let us know what you think by hitting the "Send feedback" button in Admin!
+
```
Comment (LekoArts) [inline packages/gatsby-admin/README.md:22]: Do we want to add a troubleshooting heading?

We know that there might be cases where `gatsby-config.js` has an invalid format that Admin doesn't support right now/ever.

The error message I added gives some instructions to use the "Send feedback" but maybe also place this here (in regards to this being in "Alpha" phase).

File: packages/gatsby-admin/README.md [hunk 4]
```diff
@@ -1,8 +1,28 @@
...
+You can then visit `localhost:8000/___admin` to view Admin for your Gatsby site!
+
+![Gatsby Admin homepage showing a list of installed plugins, as well as a search input to search for plugins to install](https://user-images.githubusercontent.com/7525670/95580804-36df9200-0a38-11eb-80a7-fbd847a13da1.png)
+
+When you select an installed plugin or search for a new one, you will be able to install/uninstall/configure it:
+
+![Gatsby Admin plugin page showing the README of the gatsby-plugin-sitemap, as well as an input field to configure the options for said plugin](https://user-images.githubusercontent.com/7525670/95580764-27f8df80-0a38-11eb-9f26-8a2cbbc4b07d.png)
+
+Let us know what you think by hitting the "Send feedback" button in Admin!
```
Comment (LekoArts) [inline packages/gatsby-admin/README.md:21]: Do we have an umbrella issue for issues about Admin? Sometimes the "Send feedback" input might be not enough and people will open issues. So directly route them to an issue as an alternative?

**Review by LekoArts — CHANGES_REQUESTED** [review]
Some comments :) The ENV var thing is mandatory, the other two optional

File: packages/gatsby-admin/README.md (hunk 4 above)
Comment (mxstbr) [reply packages/gatsby-admin/README.md:21]: But but feedback fish... :cry:

Comment (mxstbr) [reply packages/gatsby-admin/README.md:21]: (yes, good idea!)

**Review by LekoArts — APPROVED** [review]
LGTM 👍

**Review by ascorbic — APPROVED** [review]
Looks great!

**Pull request merged**